plugwise_usb/nodes/sense.py

```
@@ -51,21 +51,17 @@ async def load(self) -> bool:
-            _LOGGER.debug("Loading Sense node %s from cache", self._node_info.mac)
-            await self._load_from_cache()
-        else:
-            self._load_defaults()
-        self._loaded = True
-        self._setup_protocol(
-            SENSE_FIRMWARE_SUPPORT,
-            (NodeFeature.INFO, NodeFeature.SENSE),
-        )
+        _LOGGER.debug("Loading Sense node %s", self._node_info.mac)
+        if not await super().load():
+             _LOGGER.debug("Load Sense base node %s failed")     
```

Ruff (0.11.9)

Trailing whitespace

(W291)